README.md

```diff
@@ -18,35 +18,38 @@ Use the `"CI system"` component in [YouTrack](https://vimc.myjetbrains.com/youtr
 
 2. Clone this repository.
 
-## Creating the TeamCity server
-To create the TeamCity server VM and start the server, run:
+## Creating and starting the CI service
+Run (as root):
 
-    $ vagrant plugin install vagrant-persistent-storage
-    $ vagrant up montagu-ci-server
+    $ ./scripts/start-on-boot.sh
```

Comment:
needs `sudo`

Reply:
It says to run as root? I figured I'd leave up to the user how they were escalating.

Reply:
sorry, managed to miss that